src/System.Net.Http.Json/Directory.Build.props

```diff
@@ -0,0 +1,15 @@
+﻿<Project>
+  <Import Project="..\Directory.Build.props" />
+  <PropertyGroup>
+    <StrongNameKeyId>Open</StrongNameKeyId>
+    <AssemblyVersion>3.2.0.0</AssemblyVersion>
+    <PackageVersion>3.2.0</PackageVersion>
+
+    <!-- Setting Suppress Final Package Version and IsShippingAssembly properties in order to force the release build to 
+    continue emitting pre-release labels as we won't want to ship stable on this release yet. -->
+    <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
+    <BlockStable>true</BlockStable>
```

Comment:
Since this is a new branch separate from `release/3.1` we no longer need these properties to suppress the final package version and make the assembly non-shipping, as we won't mark this branch as stable until this package is RTM. 

This branch will only ship this package and will be aligned with the aspnetcore 3.2 schedule.

Reply:
Yup, will remove.